spharpy/spherical.py

```diff
@@ -248,17 +249,29 @@ def n3d_to_sn3d_norm(n):
     return 1 / np.sqrt(2 * n + 1)
 
 
-def spherical_harmonic_basis(n_max, coords):
+def spherical_harmonic_basis(
+        n_max, coordinates, normalization="n3d", channel_convention="acn",
+        condon_shortley='auto'):
     r"""
     Calculates the complex valued spherical harmonic basis matrix.
 
-    The spherical harmonic functions are fully normalized (N3D) and include the
-    Condon-Shotley phase term :math:`(-1)^m` [#]_, [#]_.
+    See [#]_ and [#]_ for details.
+    See also :py:func:`spherical_harmonic_basis_real`.
 
     .. math::
+        Y_n^m(\theta, \phi) =  CS_m N_{nm} P_{nm}(cos(\theta)) e^{im\phi}
 
-        Y_n^m(\theta, \phi) = \sqrt{\frac{2n+1}{4\pi}
-        \frac{(n-m)!}{(n+m)!}} P_n^m(\cos \theta) e^{i m \phi}
+    - :math:`n` is the degree
+    where:
+
+    - :math:`n` is the degree
+    - :math:`m` is the order
+    - :math:`P_{nm}` is the associated Legendre function
+    - :math:`N_{nm}` is the normalization term
+    - :math:`CS_m` is the Condon-Shortley phase term
+    - :math:`\theta` is the colatitude (angle from the positive z-axis)
+    - :math:`\phi` is the azimuth (angle from the positive x-axis in the
+      xy-plane)
```

Comment:
maybe we can add a link to the pyfar coordinates page here
https://pyfar.readthedocs.io/en/stable/classes/pyfar.coordinates.html (should also work using the intersphinx mapping I guess)